src/cl/lodestar/lodestar_launcher.star

```diff
@@ -22,6 +21,21 @@ DISCOVERY_PORT_NUM			= 9000
 HTTP_PORT_NUM				= 4000
 METRICS_PORT_NUM			= 8008
 
+# The min/max CPU/memory that the beacon node can use
+BEACON_MIN_CPU = 50
+BEACON_MAX_CPU = 1000
+BEACON_MIN_MEMORY = 256
+BEACON_MAX_MEMORY = 1024
+
+#  ---------------------------------- Validator client -------------------------------------
+VALIDATING_REWARDS_ACCOUNT	= "0x878705ba3f8Bc32FCf7F4CAa1A35E72AF65CF766"
```

Comment:
for later - perhaps we can pull this into a constant in `pacakage_io.constants.star`

Reply:
Fixed